### PR TITLE
Add -ip flag to specify resolve ip address

### PR DESCRIPTION
**Pull request by sosedoff**

### CODE REVIEW

**Comment by sosedoff** [comment]
Had the same issue as in #2

File: main.go [hunk 1]
```diff
@@ -34,10 +36,18 @@ func handleRequest(w dns.ResponseWriter, r *dns.Msg) {
 
 func main() {
 	var addr = flag.String("addr", "127.0.0.1:5300", "listen address")
+	var ip = flag.String("ip", "127.0.0.1", "resolve ip address")
```
Comment (robbiev) [inline main.go]: maybe change to "resolve ipv4 address"

File: main.go [hunk 2]
```diff
@@ -34,10 +36,18 @@ func handleRequest(w dns.ResponseWriter, r *dns.Msg) {
...
+
 	flag.Parse()
 
+	resolveIp = net.ParseIP(*ip)
+	if resolveIp == nil {
+		log.Fatalf("Invalid ip address: %s\n", *ip)
+	}
```
Comment (robbiev) [inline main.go:47]: Perhaps add

``` go
if resolveIp.To4() == nil {
  log.Fatalf("Invalid ipv4 address: %s\n", *ip)
}
```

File: main.go [hunk 3]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/miekg/dns"
 )
 
+var resolveIp net.IP
```
Comment (robbiev) [inline main.go]: rename to resolveIP (IP in caps, which is more common go style)

**Comment by robbiev** [comment]
Thanks! Added a few comments - mostly related to restricting to ipv4 given that the DNS resolver code only supports A records.

**Comment by sosedoff** [comment]
@robbiev changes are in

**Comment by robbiev** [comment]
Thanks, I will upload a new release binary soon.

**Pull request merged**